packages/sitecore-jss-angular/setup.js

```diff
@@ -0,0 +1,5 @@
+﻿window.process = {
+  env: {
+    NODE_ENV: 'production',
```

Comment:
Can you elaborate on this change? Do you envision all tests be performed in production context?

Reply:
Added description.
No, i updated it to `test`.